link-parent/link-stateless-spsp-receiver/src/main/java/org/interledger/link/spsp/StatelessSpspReceiverLinkSettings.java

```diff
@@ -0,0 +1,106 @@
+package org.interledger.link.spsp;
+
+import org.interledger.link.LinkId;
+import org.interledger.link.LinkSettings;
+import org.interledger.link.LinkType;
+import org.interledger.link.exceptions.LinkException;
+
+import org.immutables.value.Value;
+import org.immutables.value.Value.Derived;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * An extension of {@link LinkSettings} for ILP-over-HTTP links.
```

Comment:
This javadoc does not match the class signature. But it raises an interesting question. How will we handle the expected use case for an account that is IlpOverHttpLink on the incoming side but StatelessSpspReceiverLink on the outgoing side?

Reply:
Docs fixed.